corehq/messaging/scheduling/forms.py

```diff
@@ -664,7 +664,7 @@ def __init__(self, *args, **kwargs):
     def non_deleted_forms(self):
         return sorted(
             [form for form in self.forms if not form.is_deleted],
-            key=lambda form: form['ORDER'].value()
+            key=lambda form: int(form['ORDER'].value())
```

Comment:
Can this cast happen somewhere higher up, like in a clean method?

Reply:
good point, not sure why I originally used value(), but this is always called from a context where cleaned_data is available, so updated it: https://github.com/dimagi/commcare-hq/pull/20901/commits/231eae2afb4867fc63edcf5de453f72b092bd791